.gitignore

```diff
@@ -3,4 +3,6 @@ public/lib
 .sass-cache/
 .DS_store
 c4h_keys.txt
+mongod
+.env
```

Comment:
We can have a `.env.sample` file where we put in the env variables and corresponding dummy values. @andela-aatanda